cosmos/dbt/selector.py

```diff
@@ -43,6 +43,10 @@ def __init__(self, project_dir: Path, statement: str):
         self.other: list[str] = []
         self.load_from_statement(statement)
 
+    @property
+    def is_empty_config(self) -> bool:
```

Comment:
Since this property is within the config class, the method could be called `is_empty`.